### PR TITLE
Add explain for some plan nodes.

### PR DESCRIPTION
**Pull request by Shylock-Hg**

### CODE REVIEW

File: src/planner/Admin.cpp [hunk 1]
```diff
@@ -72,5 +72,94 @@ std::unique_ptr<cpp2::PlanNodeDescription> GetConfig::explain() const {
...
+std::unique_ptr<cpp2::PlanNodeDescription> DropUser::explain() const {
+    auto desc = DropNode::explain();
+    addDescription("username", *username_, desc.get());
+    return desc;
+}
+
+std::unique_ptr<cpp2::PlanNodeDescription> UpdateUser::explain() const {
+    auto desc = SingleDependencyNode::explain();
+    addDescription("username", *username_, desc.get());
+    addDescription("password", *password_, desc.get());
```
Comment (yixinglu) [inline src/planner/Admin.cpp:104]: Shall we return password without any mask to console ?

**Review by yixinglu — COMMENTED** [review]
Except password others LGTM

File: src/planner/Admin.cpp (hunk 1 above)
Comment (Shylock-Hg) [reply src/planner/Admin.cpp:104]: Maybe we could add note in document.

Comment (laura-ding) [reply src/planner/Admin.cpp:104]: > Shall we return password without any mask to console ?

Agree, can't return the password directly. need to modify it.

File: src/planner/Admin.cpp [hunk 2]
```diff
@@ -72,5 +72,94 @@ std::unique_ptr<cpp2::PlanNodeDescription> GetConfig::explain() const {
...
+    addDescription("username", *username_, desc.get());
+    addDescription("spaceName", *spaceName_, desc.get());
+    addDescription("role", meta::cpp2::_RoleType_VALUES_TO_NAMES.at(role_), desc.get());
+    return desc;
+}
+
+std::unique_ptr<cpp2::PlanNodeDescription> ChangePassword::explain() const {
+    auto desc = SingleDependencyNode::explain();
+    addDescription("username", *username_, desc.get());
+    addDescription("password", *password_, desc.get());
```
Comment (laura-ding) [inline src/planner/Admin.cpp:127]: ditto

File: src/planner/Admin.cpp [hunk 3]
```diff
@@ -72,5 +72,94 @@ std::unique_ptr<cpp2::PlanNodeDescription> GetConfig::explain() const {
...
+
+std::unique_ptr<cpp2::PlanNodeDescription> ListUserRoles::explain() const {
+    auto desc = SingleDependencyNode::explain();
+    addDescription("username", *username_, desc.get());
+    return desc;
+}
+
+std::unique_ptr<cpp2::PlanNodeDescription> ListRoles::explain() const {
+    auto desc = SingleDependencyNode::explain();
+    addDescription("space", util::toJson(space_), desc.get());
```
Comment (laura-ding) [inline src/planner/Admin.cpp:140]: spaceId

File: src/planner/Admin.cpp (hunk 1 above)
Comment (Shylock-Hg) [reply src/planner/Admin.cpp:104]: This won't execute in fact, so the password will not stored.

Comment (Shylock-Hg) [reply src/planner/Admin.cpp]: Had fixed.

**Review by laura-ding — APPROVED** [review]

**Review by yixinglu — APPROVED** [review]

**Pull request merged**